.NET SDK for Maintenance RP.

paul-selvaraj opened the pull request:
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

Commit:
.NET SDK for Maintenance RP.

archerzz reviewed (changes requested):
@paul-selvaraj Make sure you update the following files, according to [the guide](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec).

- `AzSDK.RP.Props`
- `.csproj`
- `AssemblyInfo.cs`

Commit:
Updated the SDK Metadata

archerzz commented on sdk/maintenance/Microsoft.Azure.Management.Maintenance/src/Properties/AssemblyInfo.cs:
```diff
@@ -15,5 +15,5 @@
-[assembly: AssemblyVersion("1.4.0.0")]
-[assembly: AssemblyFileVersion("1.4.0.0")]
+[assembly: AssemblyVersion("1.5.0.0")]
+[assembly: AssemblyFileVersion("1.5.0.0")]
```
You don't need to bump the `AsemblyVersion` if it's just a minor upgrade. See the [guide here](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec).

> AssemblyVersion: If it's a major version update (e.g. from 1.2.3.0 to 2.0.0.0), AssemblyVersion will also need to be changed

paul-selvaraj replied:
Updated

archerzz replied:
I've updated the versions. Actually for minor upgrade, bump `AssemblyFileVersion`, but don't bump `AssemblyVersion`. See https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec

> 6. Update the following information in AssemblyInfo.cs
      - AssemblyFileVersion (Must have)
      - AssemblyVersion: If it's a major version update (e.g. from 1.2.3.0 to 2.0.0.0), AssemblyVersion will also need to be changed

Commit:
Updated Assembly Version

archerzz committed:
Update AssemblyInfo.cs

when it's minor upgrade, update `AssemblyFileVersion`, but don't update `AssemblyVersion`. See: https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec

> 6. Update the following information in AssemblyInfo.cs
>   - AssemblyFileVersion (Must have)
>   - AssemblyVersion: If it's a major version update (e.g. from 1.2.3.0 to 2.0.0.0), AssemblyVersion will also need to be changed
